src/lib.rs

```diff
@@ -1,5 +1,40 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 
+//! # Hello World Example
+//!
+//! ```
+//! use rusty_v8::{Isolate, V8, Allocator, HandleScope, Script, Context, Locker, Local};
+//! use rusty_v8::platform;
+//!
+//! let platform = platform::new_default_platform();
+//! V8::initialize_platform(platform);
+//! V8::initialize();
+//!
+//! let mut create_params = Isolate::create_params();
+//! create_params.set_array_buffer_allocator(Allocator::new_default_allocator());
+//! let isolate = Isolate::new(create_params);
+//! let mut locker = Locker::new(&isolate);
+//!
+//! HandleScope::enter(&mut locker, |scope| {
```

Comment:
```
let mut hs = v8::HandleScope::new(&mut locker);
let scope = hs.enter();
```

Reply:
> ```
> let mut hs = v8::HandleScope::new(&mut locker);
> let scope = hs.enter();
> ```

 But I can't find `new` in`HandleScope`

Reply:
https://github.com/denoland/rusty_v8/blob/29fa5388f547e16596f71bfb2f0fd8e6b7021c2d/src/handle_scope.rs#L35-L54

Reply:
OK, I saw it in this project, but my example uses the version released on "create.io", which seems to be older.